Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
+
+        public Account(string name)
+        {
+            _name = name;
+        }
+
+        public Account(string name, decimal[] balance)
+        {
+            _name = name;
+            _balance = balance;
+        }
+
+        public string GetName()
+        {
+            return _name;
+        }
+
+        public decimal[] GetBalance()
+        {
+            return _balance;
+        }
+
+        public decimal GetHighestBalance()
+        {
+            return ArrayOps.FindMax(_balance);
+        }
+
+        public decimal GetLoss(int historicBalance = 2)
+        {
+            if (historicBalance < 2)
+            {
+                return default;
+            }
+            decimal previousBalance = _balance[^historicBalance];
+            decimal currentBalance = _balance[^(historicBalance - 1)];
+            return currentBalance - previousBalance;
+        }
+
+        public decimal GetCurrentBalance()
+        {
+            return _balance[^1];
+        }
+
+        public decimal GetTotalBalance()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetTotalBalance' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)